### ISSUE TITLE
📈 change code to Typescript

### ISSUE THREAD

**Issue by D3nn7** [Slynite/website#16]
**Is your feature request related to a problem? Please describe.**
If an object variables has in some case another type (e.g. Integer instead of String), then the build can immediately fail.

**Describe the solution you'd like**
Change complete JavaScript code to Typescript.

**Comment by D3nn7** [comment]
This enhancement should work on later when we have enough time for this. 

What do you think @JLS-DEV-lab?

**Comment by D3nn7** [comment]
This enhancement will be postponed for the time being until a later date.

**Comment by JLS-DEV-lab** [comment]
I agree. I'm sure we will find a better solution later on. Let us continue with travelventures and our website enhancements.

**Comment by D3nn7** [comment]
Implemented in [PR-81](https://github.com/Slynite/website/pull/81)
